### DISCUSSION TITLE
Is the QB64 extension available for Linux?

### DISCUSSION THREAD

**Discussion by Peter-Svahn** [QB64Official/vscode#133] [Q&A]
I have a Raspberry Pi 5 and was exited to see that QB64 was possible to build in VS code on it! But when searching for a QB64 extension, I cannot find any. Looks like it is only available for Window?

Do you have any plans to make it available for Debian?

**Comment by DualBrain** [comment]
Did you try adding it via the downloadable VSIX files...

https://github.com/QB64Official/vscode/tree/main/releases 

(Also note that it's not listed in the Extensions "store"; you have to install (or build?) it from the repo.)

I'm not sure what needs to be done to install it on Linux if Debian doesn't support the vsix extensions for vscode; maybe someone else could chime in here. But even without that, seems like it should be possible to copy the necessary files from the repo into the correct place for vscode/debian; says the non-Linux guy. ;-)
